api/src/main/java/org/openmrs/notification/impl/AlertServiceImpl.java

```diff
@@ -196,13 +199,10 @@ public void notifySuperUsers(String messageCode, Exception cause, Object... mess
 		alert.setSatisfiedByAny(true);
 		
 		//If there is not user creator for the alert ( because it is being created at start-up )create a user
-		//TODO switch this to use the daemon user when ticket TRUNK-120 is complete
-		if (alert.getCreator() == null) {
-			alert.setCreator(new User(1));
-		}
-		
-		// save the alert to send it to all administrators
-		Context.getAlertService().saveAlert(alert);
-		
+		if (alert.getCreator() == null) { 
+			User daemonUser = Context.getUserService().getUserByUuid(Daemon.getDaemonUserUuid());
+			alert.setCreator(daemonUser);
+		} 
+			 Context.getAlertService().saveAlert(alert);
```

Comment:
Did you run mvn clean install after these changes? The formatting does not look correct.

Reply:
@samuelmale did you see the above comment?

Reply:
@dkayiwa  gonna fix it tonight

Reply:
I just ran it for the second time @dkayiwa  but it doesn't look to have changed :(

Reply:
Though to me it looks fine

Reply:
Can you look at it from here: https://github.com/samuelmale/openmrs-core/blob/3fb475f7228b3e179b7d8479125e2ebf56fb495b/api/src/main/java/org/openmrs/notification/impl/AlertServiceImpl.java

And here: https://raw.githubusercontent.com/samuelmale/openmrs-core/3fb475f7228b3e179b7d8479125e2ebf56fb495b/api/src/main/java/org/openmrs/notification/impl/AlertServiceImpl.java

Reply:
Maybe this line https://github.com/samuelmale/openmrs-core/blob/3fb475f7228b3e179b7d8479125e2ebf56fb495b/api/src/main/java/org/openmrs/notification/impl/AlertServiceImpl.java#L206
Otherwise I don't see anything else. Just gonna manually indent it